src/frontend/src/features/layouts/components/thread-view/components/share-modal-extensions/_index.scss

```
@@ -0,0 +1,201 @@
+        &:hover {
+            background: var(--c--contextuals--background--surface--tertiary);
+        }
+    }
+
+    // ---------- "assigned" avatar badge ----------
+    // The state of an assigned user is signalled by a small brand-colored
+    // bubble in the top-right corner of their avatar, carrying a `person`
+    // glyph rendered through the Material Icons font. This satisfies RGAA
+    // 9.4 / WCAG 1.4.1 — the state is conveyed by both colour AND a
+    // distinct shape, not colour alone.
+    //
```

Stylelint (17.10.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)